consensus/votecounter/ballot.go

```diff
@@ -38,7 +40,6 @@ func (b *ballotSet[A]) add(addr *A, addrPower types.VotingPower, voteType VoteTy
 	ballot := b.ballots[*addr]
 	ballot[voteType] = true
 	b.ballots[*addr] = ballot
-
```

Comment:
The blank line is intentional, to separate between `ballots` update (3 lines) and `perVoteType` update